test/functional/api/cas/cli_messages.py

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+import re
+
+from core.test_run import TestRun
+from test_utils.output import Output
+
+
+load_inactive_core_missing = [
+    r"WARNING: Can not resolve path to core \d from cache \d\. By-id path will be shown for that "
```

Comment:
We need '\d+' instead of '\d' everywhere (IDs can have more than one digit)

Reply:
done